CHANGELOG.md

```diff
@@ -2,19 +2,92 @@
+- (Contrib) `hostmetricsreceiver`: allow configuring log pipeline to send host EntityState event ([#33927](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33927))
+- (Contrib) `windowsperfcountersreceiver`: Improve handling of non-existing instances for Windows Performance Counters ([#33815](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33815))
+  It is an expected that when querying Windows Performance Counters the targeted instances may not be present.
+  The receiver will no longer require the use of `recreate_query` to handle non-existing instances.
+  As soon as the instances are available, the receiver will start collecting metrics for them.
+  There won't be warning log messages when there are no matches for the configured instances.
+- (Contrib) `kafkareceiver`: Add settings session_timeout and heartbeat_interval to Kafka Receiver for group management facilities ([#28630](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/28630))
+- (Contrib) `vcenterreceiver`: Adds a number of default disabled vSAN metrics for Clusters. ([#33556](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33556))
+- (Contrib) `vcenterreceiver`: Adds a number of default disabled vSAN metrics for Virtual Machines. ([#33556](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33556))
+
+### 🧰 Bug fixes 🧰
+
+- (Core) `processorhelper`: update units for internal telemetry ([#10647](https://github.com/open-telemetry/opentelemetry-collector/pull/10647))
+- (Core) `confmap`: Increase the amount of recursion and URI expansions allowed in a single line ([#10712](https://github.com/open-telemetry/opentelemetry-collector/pull/10712))
+- (Core) `exporterhelper`: There is no guarantee that after the exporterhelper sends the plog/pmetric/ptrace data downstream that the data won't be mutated in some way. (e.g by the batch_sender) This mutation could result in the proceeding call to req.ItemsCount() to provide inaccurate information to be logged. ([#10033](https://github.com/open-telemetry/opentelemetry-collector/pull/10033))
+- (Core) `exporterhelper`: Update units for internal telemetry ([#10648](https://github.com/open-telemetry/opentelemetry-collector/pull/10648))
+- (Core) `receiverhelper`: Update units for internal telemetry ([#10650](https://github.com/open-telemetry/opentelemetry-collector/pull/10650))
+- (Core) `scraperhelper`: Update units for internal telemetry ([#10649](https://github.com/open-telemetry/opentelemetry-collector/pull/10649))
+- (Core) `service`: Use Command/Version to populate service name/version attributes ([#10644](https://github.com/open-telemetry/opentelemetry-collector/pull/10644))
+- (Core) `configauth`: Fix unmarshaling of authentication in HTTP servers. ([#10750](https://github.com/open-telemetry/opentelemetry-collector/pull/10750))
```

Comment:
we're not really being touched by this since it happens between 0.106.0 and 0.106.1, we can keep it here